tfhe-csprng/examples/generate.rs

```diff
@@ -4,12 +4,12 @@
 use clap::{value_parser, Arg, Command};
 #[cfg(all(target_arch = "x86_64", target_feature = "aes"))]
 use tfhe_csprng::generators::AesniRandomGenerator as ActivatedRandomGenerator;
-#[cfg(feature = "generator_aarch64_aes")]
+#[cfg(all(target_arch = "aarch64", target_feature = "neon"))]
 use tfhe_csprng::generators::NeonAesRandomGenerator as ActivatedRandomGenerator;
 use tfhe_csprng::generators::RandomGenerator;
 #[cfg(all(
     not(all(target_arch = "x86_64", target_feature = "aes")),
-    not(feature = "generator_aarch64_aes"),
+    not(all(target_arch = "aarch64", target_feature = "neon")),
```

Comment:
needs both aes and neon ?

Reply:
The implementation (before the pr) checks that both are present, so I assume that they are needed

Reply:
No i mean, the target feature here feel incomplete

Reply:
check the block ciphers implementations and their CPU feature requirements

Reply:
Oh I understand. This is the same thing than the one for x86, except that this time I mixed both features since neon is implied by aes and not the opposite. I will add both for clarity